lib/loadConfig.js

```diff
@@ -6,6 +6,8 @@ import debug from 'debug'
 import { lilconfig } from 'lilconfig'
 import YAML from 'yaml'
 
+import { resolveConfig } from './resolveConfig'
```

Comment:
I think this needs an explicit extension:

```suggestion
import { resolveConfig } from './resolveConfig.js'
```

Reply:
Hehe yes, this is why there's a smoke test that runs `node bin/lint-staged.js --version` in the CI:

```
Run node bin/lint-staged.js --version
internal/process/esm_loader.js:74

    internalBinding('errors').triggerUncaughtException(
                              ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/runner/work/lint-staged/lint-staged/lib/resolveConfig' imported from /home/runner/work/lint-staged/lint-staged/lib/loadConfig.js
    at finalizeResolution (internal/modules/esm/resolve.js:285:11)
    at moduleResolve (internal/modules/esm/resolve.js:708:10)
    at Loader.defaultResolve [as _resolve] (internal/modules/esm/resolve.js:798:11)
    at Loader.resolve (internal/modules/esm/loader.js:100:40)
    at Loader.getModuleJob (internal/modules/esm/loader.js:246:28)
    at ModuleWrap.<anonymous> (internal/modules/esm/module_job.js:47:40)
    at link (internal/modules/esm/module_job.js:46:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

Reply:
Heh, whoops, VSCode's auto-extract betrayed me. (Or I just don't have it configured right)